### ISSUE TITLE
Legends in multi line chart not accessible using keyboard navigation in uifabric/charting version '0.28.13'

### ISSUE THREAD

**Issue by anasakmsft** [OfficeDev/office-ui-fabric-react#11111]
### Environment Information

- **Package version(s)**: uifabric/charting = '0.28.13'
- **Browser and OS versions**: Chrome, Edge, Firefox/ Windows 10

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
Legends in multi line chart not accessible using keyboard navigation 

### Please provide a reproduction of the issue in a codepen:

Background: 
We see that this issue has been fixed in https://github.com/OfficeDev/office-ui-fabric-react/pull/9281. We are consuming legends in our repro and using '@uifabric/charting = '0.28.13', 

Due to business requirement we would like to stick to '0.28.13' and won't be able to upgrade to higher version yet. 

Ask from Fabric team:
We request fabric team to cherry pick the changes to the fix mentioned https://github.com/OfficeDev/office-ui-fabric-react/pull/9281  and apply in uifabric/charting: '0.28.13'
https://fabricweb.z5.web.core.windows.net/oufr/6.208.1/

#### Actual behavior:
Unable to access 'Legends' keyboard navigation 

#### Expected behavior:

Should be able to navigate Legends using keyboard Navigation

### Documentation describing expected behavior

Mentioned above in the expected behavior

**Comment by JasonGore** [comment]
Per our offline discussions, we can't change the publicly published version of 0.28.13 using the same version number. Hotfixes have to be specially consumed, typically by consuming or bundling a special branch like [these](https://github.com/OfficeDev/office-ui-fabric-react/branches/all?utf8=%E2%9C%93&query=hotfix).

**Comment by KeiraYang** [comment]
@JasonGore Can you help cherry-pick to the latest version which require Fabric 6? We cannot directly upgrade to 0.137.0 version as we're still using Fabric 6. We can upgrade the @uifabric/charting to the one which consuming Fabric 6 with this fix on. Thanks in advance!

**Comment by KeiraYang** [comment]
@JasonGore Sorry I missed the change log. @anasakmsft found the fix has been released in https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/@uifabric/charting_v0.30.1. We're trying to update package and see. Thanks!
